src/index.html

```diff
@@ -110,6 +110,28 @@
           v-1.75C12,20.05,11.78,19.82,11.5,19.82z"/>
       </g>
       </symbol>
+      <symbol id="icon-remove-hawks" viewBox="0 0 44 28">
+      <path d="M43,11.8c0-0.3-3-1.7-3-1.7c2,0.9,2.4,0.1,2.4,0.1c-0.4-0.6-3.1-1.4-3.1-1.4C39.7,8.9,40,8.4,40,8.4s-1.3-0.8-2.2-1
+        s-2.9,0.1-3.4,0.3c-0.5,0.2-0.9-0.4-1.8-0.3c-1,0.1-2.7,0.4-5.1,0.3c-2.2-0.1-2.9-0.7-3.3-0.9v0c0,0,0,0,0,0C23.8,6.7,24.1,5,23.8,4
+        c-0.3-0.9-0.5-1-0.5-1L23,2.4l-0.5,0.5c0,0-0.2,0-0.8,0.8c-0.6,0.8-0.9,2.5-1.3,2.4c0,0,0,0,0,0c-0.5,0-1.4,0.3-3.2-0.3
+        c-2.3-0.7-3.8-1.6-4.6-2c-0.9-0.5-1.4,0-1.9-0.3S8.4,2.2,7.5,2C6.5,1.9,5,2.2,5,2.2s0.1,0.6,0.6,0.7c0,0-2.8-0.2-3.4,0.3
+        c0,0,0.1,0.9,2.3,0.7c0,0-3.4,0.3-3.4,0.6c0,0.3,0.4,1.1,2.1,0.8C2.5,5.4,1.3,5.7,1,5.9C0.8,6.2,1.7,7.2,3.6,6.7
+        C3.2,6.8,1.5,7.2,1.2,7.4C1,7.7,1.9,8.1,2.3,8.2s1.2-0.1,1.2-0.1S2.7,8.6,2.9,8.9C3,9.1,4.2,9.4,4.2,9.4S4,9.8,4.1,10
+        c0.2,0.3,1.1,0.5,1.1,0.5S4.9,11,5.5,11.4c0.4,0.3,0.7,0.1,0.7,0.1s0.4,0.6,1.1,0.5c0,0,0.1,0.9,1,0.8c0,0,0.5,0.3,0.7,0.2
+        c0,0.2,0,0.5,0.1,0.8C9.3,14.2,9.9,14,9.9,14s0.3,0.9,1.1,0.5c0,0,0.1,0.6,0.6,0.4c0,0,0.4,0.4,0.7,0.1c0,0,0.3,0.9,0.9,0.2
+        c0,0,0.1,0.6,0.6,0.7c0.5,0.1,0.8-0.4,0.8-0.4s0.4,0.6,0.9,0.2c0,0,0.1,0.6,0.7,0.1c0,0,0.4,0.3,0.7-0.2c0,0,0.4,0.4,0.8-0.2
+        l0.1,0.2c0,0.1-0.1,0.1-0.1,0.1s-2.6,3.4-3.1,4.2c0,0-0.1,0.6,0.5,0.7c0,0-0.5,1.1,1,1.1c0,0-0.1,0.9,0.8,0.4c0,0,0.1,1.2,1,0.8
+        c0,0-0.1,0.9,0.8,0.4l0.5,0.4l0.6-0.2c0.8,0.7,0.9-0.1,0.9-0.1c0.8,0.7,1.2-0.4,1.2-0.4c0.8,0.7,0.9-0.1,0.9-0.1
+        c1.4,0.5,1.3-0.7,1.3-0.7c0.6,0.1,0.7-0.5,0.7-0.5c-0.1-0.9-1.5-5-1.5-5s0-0.1-0.1-0.2l0.2-0.1c0.1,0.6,0.6,0.4,0.6,0.4
+        c0.1,0.6,0.6,0.4,0.6,0.4c0.4,0.6,0.7,0.1,0.7,0.1c0.4,0.6,0.9,0.2,0.9,0.2s0.1,0.6,0.6,0.7c0.5,0.1,0.8-0.5,0.8-0.5
+        c0.3,0.9,0.9,0.2,0.9,0.2c0.2,0.3,0.7,0.1,0.7,0.1c0.4,0.4,0.8-0.2,0.8-0.2c0.6,0.7,1.2-0.1,1.2-0.1s0.4,0.4,0.7,0.1
+        c0.2-0.2,0.4-0.5,0.4-0.7c0.1,0.2,0.7,0,0.7,0c0.9,0.5,1.3-0.4,1.3-0.4c0.7,0.4,1.2-0.1,1.2-0.1s0.2,0.2,0.7,0.1
+        c0.8-0.2,0.6-0.8,0.6-0.8s0.9,0.2,1.2-0.1c0.2-0.2,0.1-0.6,0.1-0.6s1.2,0.1,1.4,0c0.3-0.2-0.3-0.9-0.3-0.9s0.7,0.4,1.1,0.5
+        c0.5,0.1,1.4,0,1.3-0.4c-0.2-0.3-1.6-1.2-2-1.5c1.7,1.1,2.9,0.5,2.7,0.1c-0.1-0.2-1.2-1-1.8-1.4C42.2,12.8,43,12.1,43,11.8z"/>
+      <g>
+        <path d="M10,19.8H3c-0.3,0-0.5,0.2-0.5,0.5v1.8c0,0.3,0.2,0.5,0.5,0.5h7c0.3,0,0.5-0.2,0.5-0.5v-1.8C10.5,20,10.3,19.8,10,19.8z"/>
+      </g>
+      </svg>
```

Comment:
This should be </symbol> otherwise we kill the definitions of the subsequent svgs.

Reply:
D'oh, don't know how I missed that. Fixed and rebased.